setup/tue-functions.bash

```diff
@@ -19,6 +19,166 @@ function _list_subdirs
     done
 }
 
+# ----------------------------------------------------------------------------------------------------
+#                                       GIT LOCAL HOUSEKEEPING
+# ----------------------------------------------------------------------------------------------------
+
+function __tue-default-branch
+{
+    local default_branch
+    default_branch=$(git remote show origin | grep HEAD | awk '{print $3}')
+    _git_remote_checkout origin "$default_branch"
+}
+
+function _tue-default-branch
+{
+    _tue-repos-do "__tue-default-branch"
+}
+
+function _tue-git-branch-clean
+{
+    # Function to remove stale branches from a git repository (which should
+    # either be the PWD or one of its parent directories). The function removes
+    # stale branches in two layers. First it removes all branches that have been
+    # merged in the remote, then it checks for unmerged branches that have been
+    # deleted from the remote and prompts for confirmation before removal. If
+    # the function is called with "--force-remove" flag, then no confirmation is asked
+
+    local force_remove=
```

Comment:
Prefer the declaration with equal sign.

Reply:
Did you mean without? Because it is with equal sign here

Reply:
without*